src/components/search-bar.tsx

```diff
@@ -1,27 +1,39 @@
 import React from 'react';
-import { Box, Input } from '@blockstack/ui';
+import { Box, Input, BoxProps } from '@blockstack/ui';
 import { MagnifyingGlass } from './icons/magnifying-glass';
 
-interface SearchBarProps {
+interface SearchBarProps extends BoxProps {
   onChange(e: React.ChangeEvent<HTMLInputElement>): void;
+  inputOffset?: string;
 }
 
-export const SearchBar = ({ onChange }: SearchBarProps) => (
-  <Box position="relative" width="100%" height="64px">
+export const SearchBar = ({ onChange, inputOffset = '50px', ...rest }: SearchBarProps) => (
+  <Box position="relative" width="100%" height="64px" borderRadius="6px" {...rest}>
     <Input
+      type="text"
       cursor="pointer"
       position="absolute"
       top={0}
       bottom={0}
       left={0}
       right={0}
-      pl="50px"
-      height="64px"
-      placeholder="Search for transactions or contracts"
-      type="text"
+      pl={inputOffset}
+      lineHeight="20px"
+      p={0}
```

Comment:
we have `pl` and then `p` which would overwrite `pl`. I'd put `pl` after `p`, or ```p={`0 0 0 ${inputOffset}`}```

Reply:
It doesn't overwrite it, but moved after anyway.